ixmp/core.py

```diff
@@ -1017,9 +1018,21 @@ def years_active(self, node, tec, yr_vtg):
         """
         return to_pylist(self._jobj.getTecActYrs(node, tec, str(yr_vtg)))
 
+    def get_meta(self, name=None):
+        return {x.getKey(): unwrap(x.getValue()) for x in
```

Comment:
should probably just use a lambda here:

```
unwrap = lambda x: x if type(value).__name__ != 'java.math.BigDecimal' else x.doubleValue()
```

is it possible to use `isinstance()` instead of `type().__name__`?

Reply:
I didn't find other way to check Java type for JPype